packages/plugin-electron-deliver-minidumps/send-minidump.js

```diff
@@ -31,9 +43,12 @@ module.exports = (net, client) => {
   }
 
   const sendMinidump = async (minidumpPath, event) => {
+    const apiKey = event && isValidApiKey(event.apiKey, client._logger)
+      ? event.apiKey
+      : client._config.apiKey
```

Comment:
I don't think this needs to be different to the core Client:

https://github.com/bugsnag/bugsnag-js/blob/34e5f635d67826638008849be60f9adeb2f523be/packages/core/client.js#L328-L329

If we do validate it, it should still be used if it isn't empty:

> If a non-empty apiKey is provided but it is not a string of exactly 32 hexadecimal characters, the notifier shall log a Warning-level validation message but use the value provided.

but if it's wrong then it will log a warning if delivery fails, so I don't think we need to validate this at all